spec/item_repository_spec.rb

```diff
@@ -45,10 +45,148 @@
                     :updated_at  => Time.now,
                     :merchant_id => 2
                   })
-      id = "263395237"
-      ir.items << i 
+      ir.add_item(i)
+
+      expect(ir.find_by_id(1)).to eq(i)
+      expect(ir.find_by_id(45)).to eq(nil)
+    end
+
+    it '#find by name' do
+      ir = ItemRepo.new
```

Comment:
We should talk with the instructors to see if we can change these instance names to things that aren't one and two-letter names. I feel like from what they've taught about convention, we should be having actual instance names (ex. ir becomes item_repo or item_repo1).

Reply:
Thanks for pointing this out! We'll make sure to update